packages/site/src/components/PaymentChannelContainer.tsx

```diff
@@ -1,21 +1,30 @@
 import { NitroRpcClient } from "@statechannels/nitro-rpc-client";
 import { PaymentChannelInfo } from "@statechannels/nitro-rpc-client/src/types";
-import { useEffect, useState } from "react";
+import { useEffect, useState, FC } from "react";
+import { makeStyles } from "tss-react/mui";
 
 import PaymentChannelList from "./PaymentChannelList";
 import PaymentChannelDetails from "./PaymentChannelDetails";
 
-type Props = {
+interface Props {
```

Comment:
Was there a particular reason for this change?

Reply:
There are few reasons why do I prefer to use interfaces:
- When defining the shape of an object or a class: Interfaces are commonly used to describe the structure of objects and classes. They convey the intention of defining contracts or blueprints for implementing classes.
- When you need declaration merging: If you want to progressively build up interfaces by merging them, interfaces are the way to go since they support declaration merging.
- When working with third-party libraries or modules: Many libraries in the TypeScript ecosystem expose their API using interfaces. It can be easier to understand and work with these APIs by using interfaces that match the library's documentation.

And interfaces align more closely with the React and TypeScript community conventions